packages/integration_test/lib/integration_test.dart

```diff
@@ -2,6 +2,20 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/// Tool for self-driving testing of Flutter code on devices and emulators.
+///
+/// It adapts `flutter_test` results into a format that is compatible with
+/// `flutter drive` and native Android instrumentation testing.
+///
+/// To use, follow the [*Integration testing* guide](https://flutter.dev/docs/testing/integration-tests).
+///
+/// See also:
+///
+///  * [Usage section in the README](https://github.com/flutter/flutter/tree/master/packages/integration_test#usage),
+///    which fully details how to use the package, how to perform Android and
+///    iOS device testing.
```

Comment:
I am not sure actually. I agree it is canonical, however, I am not sure how many people are even aware of it.

I think most people are used to using READMEs because they view packages on GitHub or Pub directly. I can only guess what the conversion rate to the API reference is.  
And in this case, I suppose it is generally not intuitive that the API reference for the package is at api.flutter.dev (it makes a lot of sense).

Reply:
Does anyone look at the flutter repo's READMEs, though? When I google [integration_test package], the top hit is to the old package on pub.dev that says it's deprecated, the second link is to [a page on flutter.dev](https://flutter.dev/docs/testing/integration-tests). That page does currently link to the README on GitHub, but that seems weird and unprofessional-looking to me. I would expect it to link to [api.flutter.dev](https://api.flutter.dev/flutter/package-integration_test_integration_test/package-integration_test_integration_test-library.html) once this PR lands.

Reply:
Do you suggest I do that in this PR? That is switch both? Do I need to update the link on flutter.dev then as well?

Reply:
@Hixie And should I link to `master-api.flutter.dev` or `api.flutter.dev`? If I use the latter, it would lead to nothing for one quarter.

Reply:
I believe we have docs linking to the readme.

/cc @johnpryan for review and feedback, since he's been working on docs for this.

Reply:
There's a lot of instructions in the [README](https://github.com/flutter/flutter/tree/master/packages/integration_test#integration_test), as long as that's okay with everyone.

We currently directly to the README from the [flutter.dev docs](https://flutter.dev/docs/testing/integration-tests) in a few places:

<img width="942" alt="Screen Shot 2021-09-09 at 12 44 19 PM" src="https://user-images.githubusercontent.com/1145719/132752441-a2c7df82-4f7b-4e7b-aac6-c084f3ca84e4.png">


<img width="598" alt="Screen Shot 2021-09-09 at 12 43 31 PM" src="https://user-images.githubusercontent.com/1145719/132752353-e08c7bf7-7088-43c1-8c92-a2014dc8dc03.png">

Reply:
@johnpryan 👍 in that case, the changes would be good as they are, right?